airbyte-integrations/connectors/source-chargebee/source_chargebee/schemas/transaction.json

```diff
@@ -66,9 +66,6 @@
     "error_code": {
       "type": ["string", "null"]
     },
-    "error-text": {
```

Comment:
Why we remove this field? will this be a breaking change?

Reply:
Exactly. Both `error-text` and `error_text` existed in the schema. I removed the incorrect one.